src/sentry/api/serializers/models/role.py

```diff
@@ -11,5 +11,6 @@ def serialize(self, obj, attrs, user, **kwargs):
             "desc": obj.desc,
             "scopes": obj.scopes,
             "is_global": obj.is_global,
+            "is_retired": obj.is_retired,
```

Comment:
Should we use camelBacked parameters like the bulk of the API does? I recognize that we will be adding a small inconsistency with `is_global` but we could also add `isGlobal` to rectify that.

Reply:
Done.